spec/spec_helper_acceptance.rb

```diff
@@ -1,82 +1,47 @@
 # frozen_string_literal: true
```

Comment:
Can you use the module synced acceptance helper?

Reply:
This is the content provided by running `msync update […]` from voxpupuli/modulesync_config.